src/app/game-state/home.service.ts

```diff
@@ -735,103 +734,14 @@ export class HomeService {
 
   }
 
-  autoBuy(){
```

Comment:
Probably biggest change here. It seemed like autobuy could be moved to its own service, especially if there was eventually a bought resource that didn't have anything to do with the home!